helix-term/src/commands.rs

```diff
@@ -417,7 +417,8 @@ impl MappableCommand {
         decrement, "Decrement",
         record_macro, "Record macro",
         replay_macro, "Replay macro",
-        command_palette, "Open command palette",
+        command_palette, "Open command pallete",
+        toggle_syntax_parser, "Toggle (on/off) syntax parser for current document",
```

Comment:
Is there any reason why this isn't `toggle_syntax`? Is it just limited to parser?

Reply:
Not really, this disables syntax parser and as a result, disables syntax highlighting. I code it does indeed make `syntax` point to `None`, so I guess it makes sense.